beacon-chain/core/helpers/signing_root.go

```diff
@@ -99,3 +100,21 @@ func computeForkDataRoot(version []byte, root []byte) ([32]byte, error) {
 	}
 	return r, nil
 }
+
+// ComputeForkDigest returns the fork for the current version and genesis validator root
+//
+// Spec pseudocode definition:
+//	def compute_fork_digest(current_version: Version, genesis_validators_root: Root) -> ForkDigest:
+//    """
+//    Return the 4-byte fork digest for the ``current_version`` and ``genesis_validators_root``.
+//    This is a digest primarily used for domain separation on the p2p layer.
+//    4-bytes suffices for practical separation of forks/chains.
+//    """
+//    return ForkDigest(compute_fork_data_root(current_version, genesis_validators_root)[:4])
+func ComputeForkDigest(version []byte, root []byte) ([4]byte, error) {
```

Comment:
As a preventive measure, perhaps add a simple test so next person has to update the test if `ComputeForkDigest` implementation changes again.

Reply:
done